src/media/css/elements--select.styl

```diff
@@ -42,10 +46,12 @@ mkt-selected:after {
     transition: transform .2s linear;
 
     +ltr() {
-        margin-left: 6px;
+        margin-left: 16px;
```

Comment:
You can remove these `margin-left` and `margin-right` lines after bumping up the `padding` above.